src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionUtilityToolFilter.java

```diff
@@ -71,6 +75,18 @@ public void doFilter(ServletRequest request, ServletResponse response,
 		M_log.debug("doFilter: Called");
 		HttpServletRequest useRequest = (HttpServletRequest) request;
 		HttpServletResponse useResponse=(HttpServletResponse)response;
+		
+		if ( BASIC_LTI_LAUNCH_REQUEST.equals(request.getParameter(LTI_MESSAGE_TYPE))) {
+			M_log.debug("new launch so invalidate any existing session");
+			if (useRequest.getSession() != null) {
+				M_log.debug("session id to invalidate: " + useRequest.getSession().getId());
```

Comment:
Logging should be done at the `info` here. Since their is chance that instructor can open 2 instances of Canvas in a same browser. I feel it might be good to see in log how often this case happen